tutorial/tutorial-gabc-03.html

```diff
@@ -174,15 +199,19 @@
 <img src="kyrie-17-15.png"/>
 </div>
 
+<p>If you don't see the difference, look particularly at the initial "K" and note where the lower diaganol branches off from the upper one.</p>
+
 <p>The best-looking free font for gregorio is a matter of taste, but <code>kpfonts</code> is also a good choice (<code>\usepackage{kpfonts}</code>).</p>
 
+<p>For a list of free fonts which are readily available for <span class="tex">T<span class="epsilson">e</span>X</span> see <a href="http://www.tug.dk/FontCatalogue/">The <span class="latex">L<span class="alpha">a</span>T<span class="epsilon">e</span>X</span> Font Catalogue.</a>  Other fonts, including most of the ones already installed on your system, can also be used, but we'll leave it up to you to explore the use of other fonts on your own.</p>
+
 <h3>Changing vertical spacing</h3>
 
-<p>To add more space between the staves, I adjusted a spacing variable by adding the following line:</p>
+<p>To add more space between the staves, I adjusted a spacing variable by adding the following line between the title and the annotation:</p>
 
 <div class="commandline">
 	<pre>
-<font color="red">\grechangedim{spaceabovelines}{1.45576 cm plus 0.36461 cm minus 0.09114 cm}{scalable}%</font>
+\grechangedim{spaceabovelines}{1.45576 cm plus 0.36461 cm minus 0.09114 cm}{scalable}%
```

Comment:
Maybe the example will be less frightening with simpler numbers like `1.45cm plus 0.4cm minus 0.9cm`?